### ISSUE TITLE
Add back explore and oculus updater buttons

### ISSUE THREAD

**Issue by alexyao2015** [Veticia/PiLauncherNext#3]
Is it possible to have the explore, accessibility and oculus updater icons added back? I know its not specifically for quest, but having them is a good convenience. 

https://github.com/lvonasek/QuestPiLauncher/blob/26911bbbcee551e0fc013e9b044968d0b81be550/Launcher/App/src/main/java/com/lvonasek/pilauncher/MainActivity.java#L434

**Comment by Veticia** [comment]
Accessibility had been moved to launcher settings. Updates, if they are anything like in pico should be self-advertising (correct me if I'm wrong). Which only leaves non-replaced Explore which I guess I can just add into the Tools category. Would that be a satisfactory solution?

**Comment by alexyao2015** [comment]
So the thing with oculus headsets is they tend to hide these settings. Those three are all accessible through launcher settings, but it's a pain to really get there. If you could add those three back somewhere, it would be great. 

Updates automatically apply on oculus headsets which is what that was for.

**Comment by Veticia** [comment]
I see. Ok then.

**Comment by Veticia** [comment]
Fixed with 3b611bebda287f59e8d82bb9a46af2aec640366e
